### ISSUE TITLE
Add Exception Handling for User-Provided IP addresses for all nodes

### ISSUE THREAD

**Issue by salmanahmed-postgresql** [stormatics/pg_cirrus#75]
As of now any IP address entered when prompted for Primary node's IP address, Standby nodes' IP address and Subnet IP address is taken as input and saved inside conf.yml file. Exception handling needs to be added so that following requirements are met before saving IP addresses to conf.yml file.

- [x] Subnet IP address should be asked first, then IP addresses of Primary and Standby nodes should be checked if they're inside the subnet
- [x] Syntax of IP addresses must be correct i.e. numbers separated by dots "."
- [x] IP addresses should be in possible IP address ranges
- [x] IP addresses should not be repeated
- [x] Nodes against each IP address must be available, can be checked by ping command
- [x] In case of wrong entries user must be asked again for IP address, but after 3 wrong entries pg_cirrus shall exit

**Comment by salmanahmed-postgresql** [comment]
Here are the logs of all exception handling added to user-provided IP addresses.
```
Enter the subnet address:
Empty input. Please enter a valid subnet address.
Enter the subnet address: 172.30.237.0/16
Invalid subnet address.
Enter the subnet address: 172.30.0.0/16


Primary PostgreSQL Server IP address: 172.30.237.10
PING 172.30.237.10 (172.30.237.10) 56(84) bytes of data.
64 bytes from 172.30.237.10: icmp_seq=1 ttl=64 time=2.19 ms

--- 172.30.237.10 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 2.186/2.186/2.186/0.000 ms


Standby 1 IP address:
Empty input. Please enter a valid IP address.
Standby 1 IP address: 1722.30.237.11
'1722.30.237.11' does not appear to be an IPv4 or IPv6 address
Standby 1 IP address: 172.30.237.11
PING 172.30.237.11 (172.30.237.11) 56(84) bytes of data.
64 bytes from 172.30.237.11: icmp_seq=1 ttl=64 time=1.36 ms

--- 172.30.237.11 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 1.358/1.358/1.358/0.000 ms
Standby 2 IP address: 172.30.237.11
IP address is already added as a Primary or Standby node.
Standby 2 IP address: 172.30.237.99
PING 172.30.237.99 (172.30.237.99) 56(84) bytes of data.
From 172.30.237.9 icmp_seq=1 Destination Host Unreachable

--- 172.30.237.99 ping statistics ---
1 packets transmitted, 0 received, +1 errors, 100% packet loss, time 0ms

Node is not reachable. Please check the IP address or node availability.
Standby 2 IP address: 172.30.237.12
PING 172.30.237.12 (172.30.237.12) 56(84) bytes of data.
64 bytes from 172.30.237.12: icmp_seq=1 ttl=64 time=1.77 ms
```

**Comment by salmanahmed-postgresql** [comment]
This issue has been fixed Pull request #83 generated against this issue. PR will be merged after review.

**Comment by salmanahmed-postgresql** [comment]
This issue has been fixed PR has been merged
